pkg/build/pipelines/go/build.yaml

```
@@ -101,8 +101,22 @@ inputs:
       If true, "go mod tidy" will run before the build
     default: "false"
 
+  ignore-untracked-files:
+    description: |
+      If true, we will provide a gitignore that ignore all untracked files.
+    default: "true"
+
 pipeline:
   - runs: |
+
+      # We sometimes have stray files in our build environment
+      # That we don't neccesarily want surfaced in the build version
+      if [ "${{inputs.ignore-untracked-files}}" == "true" ]; then
```

```suggestion
      if [ "${{inputs.ignore-untracked-files}}" = "true" ]; then
```

Pedantically speaking, string comparisons are done with `=` in shell scripts.